addons/account/tests/test_account_supplier_invoice.py

```diff
@@ -4,10 +4,26 @@
 class TestAccountSupplierInvoice(AccountingTestCase):
 
     def test_supplier_invoice(self):
-        tax = self.env['account.tax'].create({
-            'name': 'Tax 10.0',
+        tax_fixed = self.env['account.tax'].create({
```

Comment:
Well, for sure, I was kind of shocked that soo little tests have been written. I would do for each single use case one test and then a bunch of combination/integration tests.